raw_data/preprocessed/africa/kenya_english.txt

```diff
@@ -0,0 +1,325 @@
+
+(112) [1] A court may, on an application made under this Part, grant an environmental easement or an environmental conservation order subject to the provisions of this Act. [2] The object of an environmental easement is to further the principles of environmental management set out in this Act by facilitating the conservation and enhancement if the environment, in this Act referred to as the benefited environment, through the imposition of one or more obligations in respect of the use of land, in this Act referred to as the burdened land, being the land in the vicinity of the benefited environment. [3] An environmental easement may be imposed on and shall thereafter attach to the burdened land in perpetuity or for a term of years or for an equivalent interest under customary law as the court may determine. [4] Without prejudice to the general effect of subsection [2], an environmental conservation order may be imposed on burdened land so as to: [a] preserve flora and fauna; [b] preserve the quality and flow of water in a dam, lake, river or aquifer; [c] preserve any outstanding geological, physiological, ecological, archeological, or historical features of the burdened land; [d] preserve scenic view; [e] preserve open space; [f] permit persons to walk in a defined path across the burdened land; [g] preserve the natural contours and features of the burdened land; [h] prevent or restrict the scope of any activity on the burdened land which has as its object the mining and working of minerals or aggregates; [i] prevents or restrict the scope of any agricultural activity on the burdened land; [j] create and maintain works on burdened land so as to limit or prevent harm to the environment; or [k] create or maintain migration corridors for wildlife. [5] Where an environmental easement is imposed on burdened land on which any person has at the time of the imposition of the easement, any existing right or interest to the land and that environmental easement will restrict that right or interest, there shall be paid to that person, by the applicant for the environmental easement such compensation as may be determined in accordance with section 116. [6] As environmental easement may exist in gross; that is to say, the validity and enforceability of the easement shall not be dependent on the existence of a plot of land in the vicinity of the burdened land which can be benefited or, of a person with an interest in that plot of land who can be benefited by the environmental easement.
```

Comment:
Please replace `enhancement if the environment` with `enhancement of the environment`. Also, please replace `prevents or restrict` with `prevent or restrict`. Furthermore, please replace `As environmental easement` with `An environmental easement`.